host-controller/src/main/java/org/jboss/as/host/controller/DomainModelControllerService.java

```diff
@@ -897,7 +897,8 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
                     Notification notification = new Notification(ModelDescriptionConstants.BOOT_COMPLETE_NOTIFICATION, PathAddress.pathAddress(PathElement.pathElement(CORE_SERVICE, MANAGEMENT),
                             PathElement.pathElement(SERVICE, MANAGEMENT_OPERATIONS)), ControllerLogger.MGMT_OP_LOGGER.bootComplete());
                     getNotificationSupport().emit(notification);
-                    bootstrapListener.printBootStatistics();
+                    bootstrapListener.printBootStatistics(true, environment.getDomainConfigurationFile().getMainFile().getName()
+                            + ", " + environment.getHostConfigurationFile().getMainFile().getName());
                 }
             } else {
```

Comment:
What about if there is a problem? It can be considered optional, but It would be nice to get the name of the configuration file in use if your server fails to start. Keep in mind if an HC fails to start before processing the domain configuration file, you will end here with a null environment.getDomainConfigurationFile()